content/scout/integrations/environment/sysdig.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Integrate with Docker Scout using Sysdig
+description: scout, sysdig, integration, image analysis, environments, supply chain
+keywords: Integrate your runtime environments with Docker Scout using Sysdig
+---
+
+{{< include "scout-early-access.md" >}}
+
+The Sysdig integration enables Docker Scout to automatically detect the images
+you're using for your running workloads. Activating this integration gives you
+real-time insights about your security posture, and lets you compare your
+builds with what's running in production.
+
+## How it works
+
+The Sysdig Agent captures which images are being used to run your container
+workloads. Docker Scout integrates with the Sysdig API to discover the images
+in your cluster. Each Sysdig integration maps to an environment. When an image
+is detected in the cluster, that image gets assigned to the associated
+environment.
+
+The prerequisites for integrating Docker Scout with Sysdig are:
+
+- You must install the Sysdig Agent in the cluster that you want to integrate.
+  For more information, see [Sysdig docs](https://docs.sysdig.com/en/docs/installation/).
+- You must enable the runtime insights feature in Sysdig.
+
+Additionally, the images that you deploy must be analyzed by Docker Scout
+before they can be assigned to an environment. The Sysdig integration doesn't
+trigger an image analysis. To analyze images automatically, you can integrate
+Docker Scout with your registry of choice. See [Integrating container
+registries](../_index.md#container-registries).
+
+## Integrate an environment
+
+You must be an organization owner to enable this integration.
+
+1. Go to the [Sysdig
+   integrations](https://scout.docker.com/settings/integrations/ecr) page.
+2. Make sure the correct Docker organization is selected in the Dashboard
+   header.
+3. Enter a display name for this integration, and the Sysdig API token.
+4. Enter the cluster name. This is the same name as you used when deploying the
+   Sysdig Agent to the cluster.
+5. Select the region.
```

Comment:
**suggestion:** should we say anything about the connection test that happens in this step? When the user tries to click `Next` in this step, we check if the connection to the Sysdig agent is actually working and they can't move on otherwise. There's an error message we show in this situation